Added Gutenberg Like Color Picking

ElvarP opened the pull request:
Changed it so the color selection is more like the default Gutenberg color selection

![image](https://user-images.githubusercontent.com/11872349/221364239-ddbbc474-e7d1-4a50-bd05-dae5a345e631.png)

![image](https://user-images.githubusercontent.com/11872349/221364244-33fe7a53-6b40-4265-ba68-3443fdd2a408.png)

Log1x commented:
This is great! Will test ASAP.

Log1x commented:
![Screenshot](https://i.imgur.com/mA5nozS.png) 

Getting broken styles on my end. Will figure out why later if you don't get to it first.

Log1x commented:
Ah I see, the `<ul>` tag wasn't being closed.

![Screenshot](https://i.imgur.com/kQeNUM6.png)

I'd like to try to get even closer to Gutenberg's styling if possible. We can add the `Theme` text as well as work on the padding/margin between colors/on the container. 

After that, maybe see about getting it to float to the left like the Gutenberg one does.

ElvarP commented:
I'm really new to git and making pull requests btw so maybe Im doing something wrong...

I updated the styles to look a little bit more like Gutenberg as you requested, added the theme text and changed paddings/margins but I could not find a solution to make it hover to the left, had some z-index/overflow-hidden issues.

Log1x commented:
> I'm really new to git and making pull requests btw so maybe Im doing something wrong...

You're doing great. I really appreciate the contribution :)

> I updated the styles to look a little bit more like Gutenberg as you requested,

Will check this out ASAP

> but I could not find a solution to make it hover to the left, had some z-index/overflow-hidden issues.

I kinda figured this after I mentioned it.
